worker/include/Channel/ChannelRequest.hpp

```diff
@@ -23,16 +23,13 @@ namespace Channel
 	public:
 		static absl::flat_hash_map<FBS::Request::Method, const char*> method2String;
 
-	public:
-		static flatbuffers::FlatBufferBuilder bufferBuilder;
```

Comment:
This should have been thread_xxxx (I don't remember the name now) as other static members to have one per thread, right? Note: I'm not suggesting going that way.

Reply:
There is `thread_local`, but IMHO if we already have a class and methods on it, better have explicit dependency rather than thread local variables everywhere.

Reply:
Yes, that's what I meant.